### PR TITLE
Zerologon attack vector added

### PR DESCRIPTION
**Pull request by falshamri**

### CODE REVIEW

File: src-tauri/exploits/secretsdump.py [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+#   - https://moyix.blogspot.com.ar/2008/02/decrypting-lsa-secrets.html
+#   - https://moyix.blogspot.com.ar/2008/02/cached-domain-credentials.html
+#   - https://web.archive.org/web/20130901115208/www.quarkslab.com/en-blog+read+13
+#   - https://code.google.com/p/creddump/
+#   - https://lab.mediaservice.net/code/cachedump.rb
+#   - https://insecurety.net/?p=768
+#   - https://web.archive.org/web/20190717124313/http://www.beginningtoseethelight.org/ntsecurity/index.htm
+#   - https://www.exploit-db.com/docs/english/18244-active-domain-offline-hash-dump-&-forensic-analysis.pdf
+#   - https://www.passcape.com/index.php?section=blog&cmd=details&id=15
+#
```
Comment (PurpleSn0w) [inline src-tauri/exploits/secretsdump.py:49]: A Reference entry need to be added to the references page. (However still leave these code comments here.)
For guidance follow whats required from the tool you have implemented: https://apache.org/licenses/LICENSE-2.0
Make sure to link that page in the reference entry. As required in section 4.a of the agreement

File: src/components/ZeroLogon/Zerologon.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    dcName: string;
+    targetIP: string;
+    adminName: string;
+    hashes: string;
+}
+
+export function ZeroLogon() {
+    const [loading, setLoading] = useState(false);
+    const [output, setOutput] = useState("");
+    const [selectedVersion, setSelectedVersion] = useState("2.4.49");
```
Comment (Huskehhh) [inline src/components/ZeroLogon/Zerologon.tsx:17]: can remove this

File: src-tauri/exploits/secretsdump.py [hunk 3]
```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/env python
```
Comment (Huskehhh) [inline src-tauri/exploits/secretsdump.py:1]: hmmm. can we perhaps add impacket as a submodule or and clone it in our install dependencies script

**Review by Huskehhh — CHANGES_REQUESTED** [review]
Minor changes / comments - but looks really good!

File: src-tauri/exploits/secretsdump.py (hunk 3 above)
Comment (Huskehhh) [reply src-tauri/exploits/secretsdump.py:1]: apparently this now comes with kali

File: src/components/ZeroLogon/Zerologon.tsx (hunk 2 above)
Comment (Huskehhh) [reply src/components/ZeroLogon/Zerologon.tsx]: aa53fbc

File: src-tauri/exploits/secretsdump.py (hunk 3 above)
Comment (Huskehhh) [reply src-tauri/exploits/secretsdump.py:1]: made it a dep via 0177fdb

Comment (Huskehhh) [reply src-tauri/exploits/secretsdump.py]: Done 8453cfe

File: src-tauri/exploits/secretsdump.py (hunk 1 above)
Comment (Huskehhh) [reply src-tauri/exploits/secretsdump.py]: Will leave this for you - @falshamri

**Review by PurpleSn0w — APPROVED** [review]
Well done, Thanks

**Review by Huskehhh — APPROVED** [review]

**Pull request merged**